next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx

```diff
@@ -11,20 +13,32 @@ const MyApplicationHistory = ({ historyData }: MyApplicationHistoryProps) => {
     <>
       {/* Desktop */}
       <div className="hidden w-full flex-col gap-4 md:flex">
-        {historyData?.map((data, i) => (
+        {/* TODO - history data needs changes in process and on BE - until then, we just take 1 instance and present it as 'document created' (all the instances are interpreted as document created) */}
+        {historyData?.slice(-1).map((data: any, i) => (
```

Comment:
Is it `any` on purpose?

Reply:
yes, that's the TODO on top `fix the types in OpenAPI (BE)` - will add a comment here as well